packages/procaptcha/src/modules/Manager.ts

```diff
@@ -187,12 +187,23 @@ export function Manager(
                 // if the provider was already in storage, the user may have already solved some captchas but they have not been put on chain yet
                 // so contact the provider to check if this is the case
                 try {
-                    const verifyDappUserResponse = await providerApi.verifyDappUser(
+                    const signRaw = getExtension(account).signer.signRaw
+                    if (!signRaw) {
+                        throw new ProsopoEnvError('ACCOUNT.NO_POLKADOT_EXTENSION')
```

Comment:
This will throw for everybody without a polkadot extension won't it? That potentially breaks procaptcha for most users